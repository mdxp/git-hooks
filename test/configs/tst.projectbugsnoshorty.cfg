[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
[receive]
	denyCurrentBranch = ignore
[gate]
	name = gate
	notify = invalidemailaddress@abadaddress
	user-check = skip
	review-check = skip
	bug-update = skip
	mail-debug = true
[test]
	comment = 14273 git hooks should be more amenable to project gates
