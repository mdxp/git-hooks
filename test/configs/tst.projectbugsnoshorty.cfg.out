remote: changing file: the_epa[K
remote: Checks successful.[K
remote: invalidemailaddress@abadaddress [gate] push (bugs 14273)[K
remote: skipping bug update[K
